### DISCUSSION TITLE
Error running fabric javax.net.ssl.SSLHandshakeException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target

### DISCUSSION THREAD

**Discussion by escinsert** [FabricMC/fabric-api#2284] [Player support]
Hi, I triying running fabric installer for minecraft a everytime crash with this message. Any idea?   Thanks
![Captura de pantalla 2022-06-04 194217](https://user-images.githubusercontent.com/106882827/172019302-bf41e810-6939-458f-a0e0-dd4fb6815afb.png)

**Comment by warjort** [comment]
See; https://github.com/FabricMC/fabric-loader/issues/639
